packages/esm-patient-medications-app/src/add-drug-order/drug-order-form.component.tsx

```diff
@@ -51,10 +60,32 @@ const comboSchema = {
 };
 
 const schemaFields = {
-  freeTextDosage: z.string().refine((value) => value !== '', { message: 'Add free dosage note' }),
-  dosage: z.number({ invalid_type_error: 'A dosage is required' }),
-  unit: z.object({ ...comboSchema }, { invalid_type_error: 'Please select a unit' }),
-  route: z.object({ ...comboSchema }, { invalid_type_error: 'Please select a route' }),
+  // t( 'freeDosageErrorMessage', 'Add free dosage note')
+  freeTextDosage: z.string().refine((value) => value !== '', {
+    message: translateFrom(moduleName, 'freeDosageErrorMessage', 'Add free dosage note'),
+  }),
+
+  // t( 'dosageRequiredErrorMessage', 'A dosage is required' )
+  dosage: z.number({
+    invalid_type_error: translateFrom(moduleName, 'dosageRequiredErrorMessage', 'A dosage is required'),
```

Comment:
```suggestion
    invalid_type_error: translateFrom(moduleName, 'dosageRequiredErrorMessage', 'Please select a dosage'),
```

For consistency

Reply:
Actually this is a number field, the other are combo boxes, hence provide selection.